docs/feature_dynamic_macros.md

```diff
@@ -59,7 +59,7 @@ There are a number of hooks that you can use to add custom functionality and fee
 
 Note, that direction indicates which macro it is, with `1` being Macro 1, `-1` being Macro 2, and 0 being no macro. 
 
-* `dynamic_macro_record_start_user(void)` - Triggered when you start recording a macro.
+* `dynamic_macro_record_start_user(int8_t direction)` - Triggered when you start recording a macro.
```

Comment:
should include that the direction indicator is 1 and -1.

Reply:
Right, but isn't this already mentioned on the above line?